pkg/kamailio/deb/buster/kamailio.service

```diff
@@ -4,18 +4,16 @@ Wants=network-online.target
 After=network-online.target
 
 [Service]
-Type=forking
+Type=simple
```

Comment:
How will this interact with the default setting of fork=yes in the config?

Reply:
This systemd setting not kamailio process.
Interactions is not exist.

Reply:
But, doesn't this determine how systemd monitors the process after the process has started, ie. if the kamailio process forks or not?

Reply:
yes, that is determines how to monitor kamailio process.
I suggest to start kamailio with `-DD`  option.
In this case kamailio main process control own child's, and systemd monitors only kamailio main process.